_includes/contributor/2-3-submission.md

```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-info}
+**v2.3 Contribution Notice:**
+As of September 15, 2018, to keep 2.2 and 2.3 releases aligned, we accept pull requests to 2.3-develop and encourage [back ports](#porting) to 2.2-develop. We no longer accept pull requests to the v2.1 or v2.0 release lines to focus all development efforts on v2.3 and v2.2.
```

Comment:
"in an effort to keep v2.2 and v2.3 releases aligned,"

Reply:
Changed, thanks!